Tasks/AndroidSigningV3/preandroidsigning.ts

```diff
@@ -0,0 +1,27 @@
+import * as path from 'path';
+import * as secureFilesCommon from 'securefiles-common/securefiles-common';
+import * as tl from 'vsts-task-lib/task';
+
+import { ToolRunner } from 'vsts-task-lib/toolrunner';
+
+async function run() {
+    let keystoreFileId: string;
+    let secureFileHelpers: secureFilesCommon.SecureFileHelpers;
```

Comment:
This is just a 'nit', but I don't see why these  are scoped outside the if-block, and not 'const'.